Simplenote/NoticePresenter.swift

```diff
@@ -0,0 +1,170 @@
+import UIKit
+
+class NoticePresenter: KeyboardObservable {
+
+    // MARK: Properties
+    //
+    private var containerView = PassthruView(frame: .zero)
+    var noticeView: NoticeView?
+    private var noticeVariableConstraint: NSLayoutConstraint?
```

Comment:
```suggestion
    private var noticeTopConstraint: NSLayoutConstraint?
```

Nipticky: Would improve a biiit readability (all of the constraints are, technically, variable!)

Reply:
It is `variable` not because it's a var, but because at first it'a a top constraint and later it becomes bottom constraint